src/mudblazor/MudBlazor.Template.Client/Program.Main.cs

```diff
@@ -0,0 +1,26 @@
+#if (IndividualLocalAuth)
```

Comment:
This is new in the .NET 8 templates; they have a `Program.cs` and `Program.Main.cs` to support top-level statements.